src/views/guestLayout.js

```diff
@@ -8,76 +8,82 @@ import { getMessage, getLocale } from '../utils/fluent.js'
+const guestLayout = data => {
+  const siteTitle = data.meta?.title ?? getMessage('brand-fx-monitor')
+  const metaTitle = data.meta?.socialTitle ?? getMessage('brand-fx-monitor')
+  const metaDescription = data.meta?.socialDescription ?? getMessage('meta-desc-2')
 
-    <meta charset='utf-8'>
-    <meta name='viewport' content='width=320, initial-scale=1'>
-    <meta name='description' content='${getMessage('meta-desc')}'>
-    <meta name='twitter:card' content='summary_large_image'>
-    <meta name='twitter:title' content='${getMessage('brand-fx-monitor')}'>
-    <meta name='twitter:description' content='${getMessage('meta-desc')}'>
-    <meta name='twitter:image' content='${AppConstants.SERVER_URL}/images/og-image.webp'>
-    <meta property='og:title' content='${getMessage('brand-fx-monitor')}'>
-    <meta property='og:description' content='${getMessage('meta-desc')}'>
-    <meta property='og:site_name' content='${getMessage('brand-fx-monitor')}'>
-    <meta property='og:type' content='website'>
-    <meta property='og:url' content='${AppConstants.SERVER_URL}'>
-    <meta property='og:image' content='${AppConstants.SERVER_URL}/images/og-image.webp'>
+  return `
+    <!doctype html>
+    <html lang=${getLocale()}>
+      <head>
+        <title>${siteTitle}</title>
+        <style>html {display: none;}</style>
 
-    <link rel='preload' href='/fonts/Metropolis-Bold.woff2' as='font' type='font/woff2' crossorigin>
-    <link rel='preload' href='/fonts/Inter-Regular-latin.woff2' as='font' type='font/woff2' crossorigin>
-    <link rel='stylesheet' href='/css/index.css' type='text/css'>
-    <link rel='stylesheet' href='/css/partials/${data.partial.name}.css' type='text/css'>
-    <link rel='icon' href='/images/favicon-16.webp' sizes='16x16'>
-    <link rel='icon' href='/images/favicon-32.webp' sizes='32x32'>
-    <link rel='icon' href='/images/favicon-48.webp' sizes='48x48'>
-    <link rel='icon' href='/images/favicon-96.webp' sizes='96x96'>
-    <link rel='icon' href='/images/favicon-144.webp' sizes='144x144'>
-    <link rel='icon' href='/images/favicon-256.webp' sizes='256x256'>
-    <link rel='apple-touch-icon' href='/images/apple-touch-icon.webp' sizes='180x180'>
+        <meta charset='utf-8'>
+        <meta name='viewport' content='width=320, initial-scale=1'>
+        <meta name='description' content='${metaDescription}'>
+        <meta name='twitter:card' content='summary_large_image'>
+        <meta name='twitter:title' content='${metaTitle}'>
+        <meta name='twitter:description' content='${metaDescription}'>
+        <meta name='twitter:image' content='${AppConstants.SERVER_URL}/images/og-image.webp'>
+        <meta property='og:title' content='${metaTitle}'>
+        <meta property='og:description' content='${metaDescription}'>
+        <meta property='og:site_name' content='${getMessage('brand-fx-monitor')}'>
+        <meta property='og:type' content='website'>
+        <meta property='og:url' content='${AppConstants.SERVER_URL}'>
```

Comment:
Q: if we hardcode `SERVER_URL` here, does that mean anytime somebody shares on social media, it will always redirect them to the _homepage_ (and not the breach details page the user was on)?

(and I guess probably applies to src/views/mainLayout.js#L34 as well)

Reply:
You’re right! Made the URL dynamic for the pages that matter: https://github.com/mozilla/blurts-server/pull/3009/commits/2a315a211253b46b5c4452e09081595952e69dde.

Note/Question: `req._parsedOriginalUrl.pathname` is not including any query params — so it should be safe to include the result in the HTML?